frontend/packages/core/src/Feedback/error/details.tsx

```diff
@@ -0,0 +1,186 @@
+  width: "100%",
+  border: "1px solid #E7E7EA",
+  padding: "16px 16px 16px 24px",
+  borderBottomRightRadius: "8px",
+  borderTop: "unset",
+});
+
+const ErrorDetailText = styled.div({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontSize: "14px",
+});
+
+const DialogButton = styled(Button)({
+  color: "#3548D4",
+  fontWeight: 700,
+  fontSize: "14px",
+  padding: "9px 32px",
+});
+
+interface ErrorDetailsProps {
+  error: ClutchError;
+}
+
+const ErrorDetails = ({ error }: ErrorDetailsProps) => {
+  const [detailsOpen, setDetailsOpen] = React.useState(false);
+  const [expanded, setExpanded] = useControlled({
+    controlled: undefined,
+    default: false,
+    name: "Error Accordion",
+    state: "expanded",
+  });
+
+  React.useEffect(() => {
+    setDetailsOpen(false);
+  }, [error]);
+
+  const hasWrappedErrorDetails =
+    error.details.filter(detail => isClutchErrorDetails(detail)).length > 0;
+
+  const summaryIconStyle = { marginRight: "8px" };
+
+  return (
+    <>
+      <ErrorDetailDivider />
+      <Accordion elevation={0} expanded={expanded}>
+        <AccordionSummary
+          aria-controls="panel1a-content"
+          data-expanded={expanded}
+          onClick={() => setExpanded(!expanded)}
+        >
+          {!expanded ? (
+            <>
+              <ChevronRightIcon style={summaryIconStyle} /> Show more
+            </>
+          ) : (
+            <>
+              <KeyboardArrowDownIcon style={summaryIconStyle} /> Show less
+            </>
+          )}
+        </AccordionSummary>
+        <AccordionDetails>
+          <ErrorDetailContainer>
+            {hasWrappedErrorDetails && (
+              <div>
+                <ErrorDetailText style={{ color: "#0D1030" }}>
+                  The following errors were encountered:
+                </ErrorDetailText>
+                <ul>
+                  {error.details.map(detail => {
+                    // Only render Clutch Error wrapped details errors here
+                    if (isClutchErrorDetails(detail)) {
+                      const renderMax = 2;
+                      const renderItems = detail.wrapped.slice(0, renderMax);
+                      const remainingItems = detail.wrapped.length - renderMax;
+                      return (
+                        <>
+                          {renderItems.map((wrapped, idx) => {
+                            const color =
+                              grpcCodeToHttpCode(wrapped.code) >= 500 ? "#DB3615" : "#daca37";
+                            return (
+                              // eslint-disable-next-line react/no-array-index-key
+                              <ListItem key={`${idx}-${wrapped.message}`}>
+                                <ErrorDetailText style={{ fontWeight: 500 }}>
+                                  <span style={{ color }}>
+                                    {grpcCodeToText(wrapped.code)}&nbsp;
+                                  </span>
+                                  {wrapped.message}
```

Comment:
font weight of message does not match design, should not be bold

Reply:
I might be missing what you mean here but the message font weight is the same as figma
![Screen Shot 2021-03-19 at 10 33 19 AM](https://user-images.githubusercontent.com/1004789/111820411-b18dc280-889e-11eb-8bb6-7a8ac8d13e58.png)

Reply:
i think figma is having trouble figuring out the mixed weight.
![Screenshot from 2021-03-19 12-36-53](https://user-images.githubusercontent.com/4712430/111820743-db031a00-88af-11eb-977d-6dbe3a3282b1.png)

Reply:
if we add backtick or single quote monospace rendering eventually to error messages, that looks like weight 500 but the regular message text is weighted normal